core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpParser.java

```diff
@@ -312,6 +324,22 @@ public void startNextMeasurement() {
         hasNonAscii = false;
     }
 
+    private boolean completeEntity(byte endOfEntityByte, long bufHi) {
+        switch (entityHandler) {
+            case ENTITY_HANDLER_TABLE:
+                return expectTableName(endOfEntityByte, bufHi);
```

Comment:
`bufHi` is now unused in a few methods

Reply:
Removed in 7f35817193a8c111dc67871d3d08fde9cb8b7a7d